### PR TITLE
Fix bug: First Name and Last Name don't show stored values on launch.

### PR DESCRIPTION
**Pull request by elanghel**

### CODE REVIEW

File: GithubPractice/Constants.swift [hunk 1]
```diff
@@ -0,0 +1,14 @@
+//
+//  Constants.swift
+//  GithubPractice
+//
+//  Created by Elena Anghel on 3/19/21.
+//
+
+import Foundation
+struct Constants {
```
Comment (vtoloconnicov) [inline GithubPractice/Constants.swift:10]: add space before struct declaration

Comment (elanghel) [reply GithubPractice/Constants.swift:10]: ok

File: GithubPractice/ViewController.swift [hunk 2]
```diff
@@ -20,8 +20,10 @@ class ViewController: UIViewController {
     
     private let defaults = UserDefaults.standard
     
-    let jobKey = "Job"
-    let birthDayKey = "BirthDay"
+    let firstNameKey = Constants.firstNameKey
+    let lastNameKey = Constants.lastNameKey
+    let jobKey = Constants.jobKey
+    let birthDayKey = Constants.birthDayKey
```
Comment (vtoloconnicov) [inline GithubPractice/ViewController.swift]: it is a way of doing but I see no point of redeclaring these variables. you could use Constants.key right where you need it

File: GithubPractice/ViewController.swift [hunk 3]
```diff
@@ -30,25 +32,37 @@ class ViewController: UIViewController {
         lastNameField.delegate = self
         jobPositionField.delegate = self
         
+        profileImage.layer.cornerRadius = profileImage.frame.height/2
+        
+        if let fisrtName = defaults.string(forKey: firstNameKey) {
```
Comment (vtoloconnicov) [inline GithubPractice/ViewController.swift]: like here Constants.firstNameKey insead of firstNameKey

File: GithubPractice/ViewController.swift [hunk 4]
```diff
@@ -30,25 +32,37 @@ class ViewController: UIViewController {
...
+        defaults.set(lastNameField.text, forKey: lastNameKey)
         defaults.set(jobPositionField.text, forKey: jobKey)
         defaults.set(birthdayPicker.date, forKey: birthDayKey)
     }
     
     @IBAction func didClickClear(_ sender: Any) {
-        defaults.removeObject(forKey: jobKey)
-        defaults.removeObject(forKey: birthDayKey)
+        let dictionary = defaults.dictionaryRepresentation()
+            dictionary.keys.forEach { key in
```
Comment (vtoloconnicov) [inline GithubPractice/ViewController.swift]: indentaion

File: GithubPractice/ViewController.swift (hunk 3 above)
Comment (elanghel) [reply GithubPractice/ViewController.swift]: I thought the same, but then I thought that maybe the code is clearer in this way

**Review by vtoloconnicov — APPROVED** [review]

**Pull request merged**